applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h

```diff
@@ -71,6 +71,12 @@ KRATOS_DEFINE_VARIABLE( Matrix, SHELL_FORCE_GLOBAL )
 KRATOS_DEFINE_VARIABLE( Matrix, SHELL_MOMENT )
 KRATOS_DEFINE_VARIABLE( Matrix, SHELL_MOMENT_GLOBAL )
 
+// Membrane1 vairiables
+KRATOS_DEFINE_VARIABLE(double, PRESTRESS_11)
```

Comment:
sorry for the late reply and thanks a lot for the comments and advices. I am on it.